Wrap document capture error message at bounds of file input

aduth opened the pull request:
**Why**: Per design feedback.

Before|After
---|---
![before](https://user-images.githubusercontent.com/1779930/96734294-640d3800-1388-11eb-8bb1-e7eab423012b.png)|![after](https://user-images.githubusercontent.com/1779930/96734065-23152380-1388-11eb-9390-32cc84a6f708.png)

**Implementation Notes:**

This turned out to require a messy cascade of unrelated revisions, triggered by a desire to adopt CSS imported via React components (introduced in #4260). I've made a point to describe the purpose of each of these changes in detailed commit history. Ultimately, it's made worse by outdated and implicit dependencies (see also https://github.com/18F/identity-style-guide/pull/159).

I think these are worth working through in order to benefit from advantages of granular CSS bundles, but for this purpose of this requirement, it might be best to either...

- Limit to contain changes to the main application CSS bundle (`app/assets/stylesheets/components/_file-input.scss`)
- Or: Split out some of the unrelated changes to separate pull requests
- Or: Wait for https://github.com/18F/identity-style-guide/pull/159, which will simplify some of the more unfortunate concessions

zachmargolis commented on app/javascript/packages/document-capture/components/acuant-capture.scss:
```diff
@@ -0,0 +1,37 @@
+    &::after {
+      content: '';
+      display: block;
+      // 2.125" x 3.375" are common standard ID dimensions
+      padding-bottom: ((2.125 / 3.375) * 100) + unquote('%');
```
I know this just got copy-pasted from the other file, but since we have the same comment and line twice, is it worth taking advantage of SASS/SCSS and making a variable for this? `$common-id-card-ratio` or something

aduth replied:
Yeah, makes sense.

Could also use a placeholder or mixin to apply the padding (and maybe `display: block`), since unless we're willing to assign the ratio as the percentage value, we'd still have to do some math + concatenation:

```scss
padding-bottom: ($common-id-card-ratio * 100) + unquote('%');
```

aduth replied:
Updated to use SASS placeholder in 2df9969.

zachmargolis reviewed (approved):
LGTM

anniehirshman-gsa reviewed (commented):
Visually, screenshot LGTM!  

Note: I updated the text in the error inventory, which will avoid the orphan for this specific message in the screenshot.

aduth commented:
> Note: I updated the text in the error inventory, which will avoid the orphan for this specific message in the screenshot.

For context, the updated messaging:

>This file type is not accepted, please choose a JPG, PNG, HEIC, BMP, or TIFF file.

Annie and I chatted briefly about this, since part of my intention with these changes was to refactor the component slightly to allow it to be more reusable. A message like this is challenging, since not all usages of a file input component would need these specific file types.

What we landed on was to allow the text to be customized where it's used, just like with other texts (hint, etc), so that we can still tailor the message appropriately to the hint.

I've updated this in 669ff6d and 32cadc2. Here's how the customized message looks:

![image](https://user-images.githubusercontent.com/1779930/96887163-515f3580-1452-11eb-88b9-cabc3808d397.png)

Since we'd still want a fallback text, I've left this from the initial implementation. Though based on @anniehirshman-gsa 's feedback, shortened it slightly in 06cff29  to allow it to fit on one line:

![image](https://user-images.githubusercontent.com/1779930/96887052-2ecd1c80-1452-11eb-82c1-a1c08623a6d7.png)

anniehirshman-gsa commented on config/locales/errors/en.yml:
```diff
@@ -72,7 +74,7 @@ en:
         You can also click on Start Over and choose to use your computer instead.
     file_input:
-      invalid_type: This file type is not accepted, please choose another file.
+      invalid_type: This file type is not accepted, please choose another.
```
```suggestion
      invalid_type: This file type is not accepted, please try again.
```

anniehirshman-gsa commented on config/locales/errors/es.yml:
```diff
@@ -73,7 +75,7 @@ es:
         También puedes hacer clic en comenzar de nuevo y elegir usar tu computadora.
     file_input:
-      invalid_type: Este tipo de archivo no es aceptado, elija otro archivo.
+      invalid_type: Este tipo de archivo no es aceptado, elija otro.
```
```suggestion
      invalid_type: Este tipo de archivo no es aceptado, inténtalo de nuevo.
```

anniehirshman-gsa commented on config/locales/errors/fr.yml:
```diff
@@ -74,8 +76,7 @@ fr:
     file_input:
-      invalid_type: Ce type de fichier n'est pas accepté, veuillez choisir un autre
-        fichier.
+      invalid_type: Ce type de fichier n'est pas accepté, veuillez en choisir un autre.
```
```suggestion
      invalid_type: Ce type de fichier n'est pas accepté, veuillez réessayer.
```

anniehirshman-gsa reviewed (changes requested):
Shortened for consistency with other error messages. Thanks!

aduth commented:
Thanks for catching that mistake @anniehirshman-gsa . Updated in 28ea67e.

anniehirshman-gsa reviewed (approved):
LGTM
